source/Halibut.Tests/Util/SerilogLoggerBuilder.cs

```diff
@@ -1,19 +1,68 @@
+using System;
+using System.IO;
+using NUnit.Framework;
 using Serilog;
+using Serilog.Core;
+using Serilog.Events;
 using Serilog.Formatting.Display;
 using Serilog.Sinks.NUnit;
 
 namespace Halibut.Tests.Util
 {
     public class SerilogLoggerBuilder
     {
-        const string OutputTemplate = "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level}] {Message}{NewLine}{Exception}";
+        const string OutputTemplate = "{Timestamp:HH:mm:ss.fff zzz} {Message}{NewLine}{Exception}";
         
         public ILogger Build()
         {
+            // In teamcity we need to know what test the log is for, since we can find hung builds and only have a single file containing all log messages.
+            var testName = "";
+            if (TeamCityDetection.IsRunningInTeamCity())
+            {
+                testName = "[{TestName}] ";
+            }
+            
+            var outputTemplate = 
+                testName
+                + "{Message}{NewLine}{Exception}";
+            
             return new LoggerConfiguration()
                 .MinimumLevel.Debug()
-                .WriteTo.Sink(new NUnitSink(new MessageTemplateTextFormatter(OutputTemplate)))
+                .WriteTo.Sink(new NonProgressNUnitSink(new MessageTemplateTextFormatter(outputTemplate)))
                 .CreateLogger();
         }
+        
+        public class NonProgressNUnitSink : ILogEventSink
+        {
+            private readonly MessageTemplateTextFormatter _formatter;
+
+            public NonProgressNUnitSink(MessageTemplateTextFormatter formatter) => _formatter = formatter != null ? formatter : throw new ArgumentNullException(nameof(formatter));
+
+            public void Emit(Serilog.Events.LogEvent logEvent)
+            {
+                if (logEvent == null)
```

Comment:
Style police Nits:

All places I've worked don't like new lines after 'if' without curly braces.

I'd prefer curly braces, or put them on the same line without curly braces.


Also, there's a missing 'var' below (consistency)